mails/themes/modern_mjml/core/order_conf.mjml.twig

```diff
@@ -117,6 +117,7 @@
           {total_discounts}
         </td>
       </tr>
+      <mj-addif condition="giftWrapping == 1">
```

Comment:
I'm not sure about the name `mj-addif`, it feels like we're using a MJML feature when we don't. Why not simply use
```suggestion
      <if condition="giftWrapping == 1">
```
or even
```suggestion
      <twig-if condition="giftWrapping == 1">
```

Reply:
Fair point!